examples/reti/client.py

```diff
@@ -0,0 +1,4019 @@
s:224", "teal": 629}, {"pc": [597, 598], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 633}, {"pc": [599, 600], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 634}, {"pc": [601, 602, 603], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 635}, {"pc": [604], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 636}, {"pc": [605], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 637}, {"pc": [606], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 638}, {"pc": [607, 608], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 639}, {"pc": [609], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 640}, {"pc": [610], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 641}, {"pc": [611, 612], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 642}, {"pc": [613], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 643}, {"pc": [614, 615], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 644}, {"pc": [616, 617, 618], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 645}, {"pc": [619], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 646}, {"pc": [620], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 647}, {"pc": [621], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 648}, {"pc": [622, 623], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 649}, {"pc": [624], "source": "contracts/validatorRegistry.algo.ts:226", "teal": 650}, {"pc": [625], "source": "contracts/validatorRegistry.algo.ts:224", "teal": 651}, {"pc": [626], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 656}, {"pc": [627, 628, 629], "source": "contracts/validatorRegistry.algo.ts:237", "teal": 659}, {"pc": [630], "source": "contracts/validatorRegistry.algo.ts:237", "teal": 660}, {"pc": [631, 632, 633], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 663}, {"pc": [634], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 664}, {"pc": [635], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 665}, {"pc": [636], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 666}, {"pc": [637], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 667}, {"pc": [638], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 668}, {"pc": [639, 640, 641], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 678}, {"pc": [642], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 681}, {"pc": [643, 644], "source": "contracts/validatorRegistry.algo.ts:236", "teal": 682}, {"pc": [645, 646], "source": "contracts/validatorRegistry.algo.ts:238", "teal": 686}, {"pc": [647, 648], "source": "contracts/validatorRegistry.algo.ts:238", "teal": 687}, {"pc": [649], "source": "contracts/validatorRegistry.algo.ts:238", "teal": 688}, {"pc": [650, 651], "source": "contracts/validatorRegistry.algo.ts:238", "teal": 689}, {"pc": [652], "source": "contracts/validato
```

Comment:
TS generates this as a tuple rather (no list[int]). Is there any reason to do it like you have, as I've noticed all tuples are generated like this.

Reply:
@neilcampbell need more context here, not sure what you are referring to exactly - is it some field in the serialized json app spec?